api/src/main/java/org/apache/brooklyn/api/mgmt/rebind/mementos/BrooklynMementoRawData.java

```diff
@@ -188,14 +211,22 @@ public String getBrooklynVersion() {
         return Collections.unmodifiableMap(catalogItems);
     }
     
+    public Map<String, String> getBundles() {
+        return Collections.unmodifiableMap(bundles);
+    }
+    
+    public Map<String, ByteSource> getBundleJars() {
+        return Collections.unmodifiableMap(bundleJars);
+    }
+    
     // to handle reset catalog
     @Beta
     public void clearCatalogItems() {
         catalogItems.clear();
     }
     
     public boolean isEmpty() {
-        return planeId == null && entities.isEmpty() && locations.isEmpty() && policies.isEmpty() && enrichers.isEmpty() && feeds.isEmpty() && catalogItems.isEmpty();
+        return planeId == null && entities.isEmpty() && locations.isEmpty() && policies.isEmpty() && enrichers.isEmpty() && feeds.isEmpty() && catalogItems.isEmpty() && bundles.isEmpty();
```

Comment:
would be misleading -- better would be an assertion bundleJars isn't empty if the bundles is, but that's also not worth it i think